packages/nimble-tokens/dist/icons/svg/asterisk_5x5.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="Layer_1"
+	xmlns="http://www.w3.org/2000/svg" viewBox="0 0 5 5">
```

Comment:
The fact that it's 5x5 instead of 16x16 is a bit different. I guess we could debate whether the default size of the element should respect that. But since that'd be new behavior and no one is around this week to debate it, I'll leave it as-is and we can change it later if needed.